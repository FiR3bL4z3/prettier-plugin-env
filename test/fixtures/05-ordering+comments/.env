B=BB
# Comment for A
A=AA
# Comment for C
# Another comment for C
C=CC
# Random trailing comment