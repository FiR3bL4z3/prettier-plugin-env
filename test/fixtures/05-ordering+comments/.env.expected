# Comment for A
A = AA
B = BB
# Comment for C
# Another comment for C
C = CC
# Random trailing comment